### ISSUE TITLE
Swipeable cards a11y

### ISSUE THREAD

**Issue by robdodson** [GoogleChrome/ui-element-samples#18]
The cards present an interesting challenge because there isn't a great analog for them in existing browser semantics.

Some things I was considering...

Can the cards be made out of a ul and li instead of divs? That way the screen reader would announce how many cards are in the list. Alternatively I think you could give the parent div a role of `list` and the children a role of `listitem`.

Can the cards be made focusable. Maybe adding a hint with an aria-label that they can be deleted either by swiping or pressing the esc key.

**Comment by paullewis** [comment]
Makes sense to update to a list, I think, yep. And I think we maybe want some kind of dismissal button on each card that you can tab between that would dismiss that card. And yeah, maybe adding escape key bits to it would make sense.

**Comment by robdodson** [comment]
I guess we could make a dismissal button that only appears if keyboard focused. I want to send a message to our a11y mailing list to figure out how other google products handle accessible swipeable cards

**Comment by oldergod** [comment]
Why not the delete key to dismiss cards? Is the escape key use a lot for this? I'd rather see the latter as a defocus functionality.

**Comment by robdodson** [comment]
@oldergod I was considering the delete key but I think it can also be used as a back button for the browser. Escape is typically used for dismissing things dialogs but I agree it's also non-obvious to use it in this case.
